src/content/boundary/product-landing.json

```diff
@@ -26,7 +26,7 @@
 			},
 			{
 				"icon": "hcp",
-				"text": "Boundary OSS Quick Start",
+				"text": "Boundary Community Quick Start",
```

Comment:
This is associated with an actual collection title so I'd be hesitant to make this change.

Reply:
ok, removed it for now.